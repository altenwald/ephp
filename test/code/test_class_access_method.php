<?php

class A {

    private function no_call_me() {
        print "I told you!\n";
    }

}

$a = new A();
$a->no_call_me();

