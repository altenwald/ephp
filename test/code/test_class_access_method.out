
Fatal error: Call to private method A::no_call_me() in {{CWD}}/test/code/test_class_access_method.php on line 12
